Kbs.Wpf/Course/Read/Details/ReadDetailsCourseViewModel.cs

```diff
@@ -25,7 +25,7 @@ public int Id
         } 
     }
 
-    public string IdFormatted => $"Parcours #{Id}";
+    public string IdFormatted => $"Parcour #{Id}";
```

Comment:
het is gewoon 'Parcours' hoor, niet 'Parcour'
Enkelvoud is 'Parcours' en meervoud is 'Parcoursen'
Source: https://www.vandale.nl/gratis-woordenboek/nederlands/betekenis/parcour